src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="Ford Sync RAPI" version="4.3" date="2017-03-29">
+<interface name="Ford Sync RAPI" version="4.3.1" date="2017-04-20">

```

Comment:
This should force a minor version increase. Since checking minor version 3 will not yield any useful results.